pandas/tests/series/methods/test_interpolate.py

```diff
@@ -78,6 +78,40 @@ def interp_methods_ind(request):
 
 
 class TestSeriesInterpolateData:
+    def test_interpolate_datetimelike_values(self):
```

Comment:
Do we have DataFrame.interpolate tests with NaT?

Reply:
i dont think so.  before long ill do a pass to parametrize all of these over frame_or_series

Reply:
good suggestion, turns out it was broken with DataFrame.  fixed.